lib/tsc-watch.js

```diff
@@ -193,7 +213,7 @@ const Signal = {
 
 nodeCleanup((_exitCode, signal) => {
   tscProcess.kill(signal);
-  killProcesses(true).then(() => process.exit());
+  killProcesses(true).catch(silentlyHandleCancellation).finally(() => process.exit());
```

Comment:
The change from `then` to `finally` isn't strictly necessary to fix this specific issue, but it could fix other potential issues  if something went wrong in `killProcesses()`.